pkg/manifests/manifests_test.go

```diff
@@ -4322,9 +4322,19 @@ func TestPodDisruptionBudget(t *testing.T) {
 }
 
 func TestPrometheusOperatorUserWorkloadConfiguration(t *testing.T) {
-	c, err := NewConfigFromString(`
-enableUserWorkload: true
-`, false)
+	c := NewDefaultConfig()
+	uwc, err := NewUserConfigFromString(`prometheusOperator:
+  enableUserWorkload: true
```

Comment:
`enableUserWorkload: true` is not needed, since this func is test UWM prometheus-operator

Reply:
right, thanks